packages/sak-fagsak-profil/src/FagsakProfilSakIndex.stories.tsx

```diff
@@ -24,7 +24,7 @@ const Template: StoryFn = () => (
         kodeverk: FAGSAK_YTELSE_KODEVERK,
         navn: 'Foreldrepenger',
       }}
-      fagsakMarkeringTekst="Næring"
+      fagsakMarkeringTekster={['Næring']}
```

Comment:
Her burde det legges inn flere tags for storiens del

Reply:
Done